docs/source/user-guide/cheatsheet.rst

```diff
@@ -1,22 +1,40 @@
-===========
-Cheat sheet
-===========
+==========
+Cheatsheet
+==========
 
-See the :download:`conda cheatsheet <conda-cheatsheet.pdf>` PDF
-(1 MB) for a single-page summary of the most important
-information about using conda (link always points to the latest version).
+Refer to the conda cheatsheet for basic commands for creating and managing environments, installing packages, and importing and exporting environments.
 
-Versions
-========
+.. csv-table:: Quickstart
```

Comment:
I think this page would look better if we had the headers as normal sphinx headers rather than using the csv-table plugin.

For example, that plugin center aligns the headers, which looks a little weird. If we make these secondary headers, then they will also show up on the navigation sidebar.

Reply:
I concur, see my suggestions below

Reply:
Definitely!